src/potr.rs

```diff
@@ -173,7 +177,36 @@ impl Potr {
                 tracing::debug!("Skip code block message: {}", message.msgid());
                 return false;
             }
-        } else if self.config.skip_text {
+        }
+        
+        // len == 1, do not translate.
+        else if message.msgid().len() == 1 {
+            return false;
+        }
+
+        // formula block, starts_with "$$"
+        else if message.msgid().starts_with("$$") {
+            if self.config.skip_formula_blocks {
+                tracing::debug!("Skip $$ prefixed message(skip formula): {}", message.msgid());
+                return false;
+            }
+        } 
+        // image file, starts_with "![]("
+        else if message.msgid().starts_with("![](") {
+            if self.config.skip_md_image_file {
```

Comment:
I think we only need the second one for skipping the images. : D

Reply:
But if there is only this condition(`.starts_with("![]("`),  there is no way to handle the situation:
```bash
 ![image_name](link)
```
So I think it is necessary to retain these 2 conditions, what do you think?